src/server/game/Entities/Item/ItemPrototype.h

```diff
@@ -737,6 +737,41 @@ struct ItemTemplate
         return std::max<float>(0.f, itemLevel);
     }
 
+    uint32 GetSkill() const
```

Comment:
unrelated to this PR and related to the existing code too: what is the point of placing all the logic in the header ?